calcom/cal.com
# When rescheduling only change calendar associated with booking

## What does this PR do?



Previously when rescheduling a booking, the event manager would try and update all connected calendars. This triggers the alarm system since the booking is only created on one calendar. This PR aims to fix that by only finding the calendar that the booking was created on initially.

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Connect two calendars of the same type (I connected two google calendars)
- Create a booking
- Reschedule that booking now
- Go into the the DB and finding the calendar booking reference. Delete the credential ID
- Reschedule that booking again. It should be rescheduled on your calendar

## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
